### ISSUE TITLE
Otaa support

### ISSUE THREAD

**Issue by DiederikRhee** [Ideetron/RFM95W_Nexus#4]
A really nice and small libary! But does anyone know's how to implement OTAA with this libary?

**Comment by JeroenIoT** [comment]
Hey Diederik, do you know if OTAA was ever implemented? I like the simplicity of the library  but need OTAA as well ... if not on the RFM95 there are other options to get OTAA libs

**Comment by GerbendH** [comment]
OTAA is not yet implemented in this library. It is planned to add OTAA as soon as possible

**Comment by nehmeZ** [comment]
Hello, Has OTAA been implemented in your library ?

**Comment by barthiddink** [comment]
No, not in this Lib. Please use the Nexus_LoRaWAN sketch. (Browse Ideetron root)
